manager-bundle/src/Command/MaintenanceModeCommand.php

```diff
@@ -43,26 +43,60 @@ protected function configure(): void
     {
         $this
             ->setName('contao:maintenance-mode')
-            ->addArgument('state', InputArgument::REQUIRED, 'Use "enable" to enable and "disable" to disable the maintenance mode. If the state is already the desired one, nothing happens. You can also use "on" and "off".')
+            ->setAliases(['lexik:maintenance:lock', 'lexik:maintenance:unlock'])
```

Comment:
What happens if someone has `LexikMaintenanceBundle` still installed?

Reply:
no idea what Symfony would do in that case, but add added handling for it in 85e2d1ccfc6b37e9a49231ddf15742257465a2d4

Reply:
Nice 👍